apps/fipsinstall.c

```diff
@@ -376,7 +376,7 @@ int fipsinstall_main(int argc, char **argv)
 
     /* No extra arguments. */
     argc = opt_num_rest();
-    if (argc != 0)
+    if (argc != 0 || (verify && in_fname == NULL))
```

Comment:
Not sure why we couldnt use stdin if in_fname is NULL.. app_load_config would use that if the filename is NULL inside the verify()

Reply:
I wasn't positive that `app_load_config` did the right thing with NULL vs "".
Fixup commit pushed to add stdin.
And wow, that app bio-open/config stuff has gotten ugly.

Reply:
when i tried to change this, something in the `req` test hung, so I reverted.  (Like I said, that code is now ugly)

Reply:
That seems rather weird since the change should have been local to fipsinstall and had no effect elsewhere..

Reply:
> That seems rather weird since the change should have been local to fipsinstall and had no effect elsewhere..

No, the change was to `app_load_config_with_bio_on_thursday`, or something like that.  Common code with special semantics that broke something else :(